src/planeta/utils.js

```diff
@@ -69,10 +79,25 @@ async function findPassportOutput(plasma, address, color, value) {
   return passports.filter(p => p.output.value === value)[0];
 }
 
-export async function startHandshake(web3, passport, privateKey) {
+export async function startHandshake(
+  web3,
+  passport,
+  privateKey,
+  strategy = "collaborate"
+) {
   // For now we hardcode the CO₂ emitted by 8 Gt, why 8 Gt? Answer here:
   // https://docs.google.com/spreadsheets/d/1chB4P7C594ABGn2u3VQb73t2F_0YPq26OHGJt0ZuME0/edit#gid=0
-  const passportDataAfter = updateCO2(passport.output.data, "8000");
+  let passportDataAfter;
+
+  if (strategy === "collaborate") {
+    passportDataAfter = updateCO2(passport.output.data, "400");
```

Comment:
Based on the body of updateCO2 method, I would call it differently - maybe `calculateUpdatedCO2"? 🤷‍♂️ 
I had assumption that it actually changes the value of CO2 in the passport, but it only computes new value.